ng-dev/utils/git/git-client.ts

```diff
@@ -197,6 +197,11 @@ export class GitClient {
     );
   }
 
+  /** Adds the provided files to the staging area. */
+  stageFiles(files: string[], options: GitCommandRunOptions = {}): SpawnSyncReturns<string> {
```

Comment:
This really a nit: I'd prefer to not add this here since it seems easy enough to invoke where needed. I don't think we intend to build a nice API around git, rather try to share commonly used, rather complicated code here. Not feeling strongly though (the method just limits the flexibility of the add command)

Reply:
I'm probably unhappy about this because we have other git add places already that are now left behind.

Reply:
Sure I understand :slightly_smiling_face::+1:

I am not that familiar with this codebase, but I guess it could be nice to move some functionality in the git client, in case you'd like that just let me know and I would really like to try to move some stuff in the git client (consistently, without leaving stuff behind)

(although as you said these are pretty simple commands and `git`'s api as far as I can tell is pretty static, so there may not be too much benefit in that :shrug:)

Reply:
I think I agree with Paul here, I would prefer we just put the staging logic at the call site.

If we were to create a generalized method here, I would prefer it to be `add()` since thats the actual thing that's called.

Reply:
yeah totally agree with the naming of course :slightly_smiling_face::+1:  (but yeah since it doesn't seem to be worth it anyways never mind :sweat_smile:)